content/pt/docs/guias/TOKEN-DE-ACESSO/Autenticacao/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Autenticação"
+linkTitle: "Autenticação"
+date: 2021-06-28T09:00:00-03:00
+lastmod: 2021-06-28T09:00:00-03:00
+weight: 4
+draft: false
+description: >
+
+---
+<br>
+
+Para realizar o processo de Autenticação, você deverá enviar uma requisição ao nosso servidor contendo um JWT. O JWT enviado deverá conter as credenciais da sua aplicação ([ClientID](/docs/guias/token-de-acesso/cadastro-da-aplicacao/)) e deverá ser assinado com a chave privada. Ao receber o request, faremos o processo de validação do token enviado, pois já teremos recebido e armazenado a sua chave pública.
+
+Uma vez que o token seja válido e tenha sido autenticado em nosso servidor, enviaremos como resposta o Access Token, ou seja, o seu Token de Acesso. <br><br>
+
+##### <ins>Padrões utilizados na Autenticação</ins>
+
+Para facilitar ao máximo a integração com nossa API, escolhemos padrões estabelecidos no mercado para realização do processo de autenticação - descritos abaixo. 
+
+**OAuth 2.0** é um padrão de autorização que vai permitir ao usuário conceder acesso limitado a recursos na Conta Stone para Aplicações sem a necessidade de expor suas credenciais. Isso significa que o cliente não precisará compartilhar login e senha com ninguém. 
+
+**OpenID Connect** é uma camada de identidade construída em cima do OAuth 2.0 que permite a fácil verificação da identidade do usuário, bem como a capacidade de obter informações básicas de perfil do provedor de identidade - no nosso caso, a Conta Stone. <br><br>
+
+##### <ins>Especificações do JWT para realizar autenticação</ins>
+
+A [chamada]() é realizada através do método POST com as informações abaixo:
```

Comment:
link is pointing to nowhere, it should behave like that?

Reply:
Oops, I updated!